roles/letsencrypt/tasks/dns-challenge-autodns.yml

```diff
@@ -25,7 +25,7 @@
             "origin": "{{ item }}",
             "resourceRecordsAdd": [
               {
-                "name": "_acme-challenge.{{ item }}",
+                "name": "_acme-challenge.{{ item | replace ('*.','') }}",
```

Comment:
are you sure this is right? normally in a RegEx "." matches for any character and "*" is a modifier for any count. if you want to strip a "." from the end of the string I would expect a RegEx like "\.$"

Reply:
I just noticed, this is not `regex_replace` but the jinja `replace` (https://jinja.palletsprojects.com/en/2.11.x/templates/#replace) with this the "*" makes no sense to me and the "." would replace all occurences of ".".

Reply:
I tested it with multiple domains and even a wildcard of a subdomain. it correctly did strip the "*." From the entry and kept all other occurrences of ".". 
Tested domains where looking like this:
- testcert.example.com
- *.example.com
- *.testcert.example.com

In the linked jinja document nothing about regex is mentioned. Also I couldn't find any other document stating otherwise. Besides the regex here would of course be wrong as I first set a quantifier and after this set the character to match. Seems to me that this does work as I want it to :) 
Do you know of any better filter to achieve this?

Reply:
well it seems I misread your commit message and I thought this would handle the case, where there is an additional "." in the DNS name. So everything is ok. And replace is the right way to go here.